src/waveform/renderers/waveformwidgetrenderer.h

```diff
@@ -93,6 +96,9 @@ class WaveformWidgetRenderer {
     }
 
     void setTrack(TrackPointer track);
+    void setMarkLabelBoundaries(QMap<WaveformMarkPointer, int> markLabelOffsets) {
+        m_markLabelOffsets = markLabelOffsets;
+    }
```

Comment:
"Offset" implies that these are somehow shifted from another position.
```suggestion
    void setMarkLabelPositions(QMap<WaveformMarkPointer, int> markLabelPositions) {
        m_markLabelPositions = markLabelPositions;
    }
```